packages/tests/src/utils/cleanHelper.ts

```diff
@@ -573,7 +573,7 @@ export async function createResourceGroup(
 
 export async function createResourceGroupByName(
   name: string,
-  location = "eastus"
```

Comment:
why change here?

Reply:
reverted the change for default region and set the region in provision function. TTK provision also use westus.
![image](https://github.com/OfficeDev/TeamsFx/assets/49181894/73563204-8d84-41f4-b1b7-8c71267cd255)